058-PowerPlatformBasics/Coach/Solution-01.md

```diff
@@ -0,0 +1,16 @@
+# Challenge 01 - Challenge Setup - Coach's Guide 
+
+[< Previous Solution](./Solution-00.md) - **[Home](./README.md)** - [Next Solution >](./Solution-02.md)
```

Comment:
this is a navigation link to "Solution-00", which is typically prerequisites. your WTH doesn't have a solution 00, you put the setup in challenge/solution 01, either re-number all of your solutions & challenges (lots of work :)) or remove this navigation link to non-existent files

something like:
`**[Home](./README.md)** - [Next Solution >](./Solution-02.md)`